docker/bin/certify.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -eu
+
+DOCKER_BIN="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+export PASS="abcdefgh"
+source ${DOCKER_BIN}/../.env
+
+if [[ -f ${TLS}/ca-cert ]]; then
+    echo "${TLS}/ca-cert found; skipping certificate generation.."
+    exit 0
+fi
+
+# Clean up old certs, warning this is
```

Comment:
sentence is cut off, but I think this should be removed anyway since it is commented out